src/app/donation-thanks/donation-thanks.component.ts

```diff
@@ -195,9 +238,7 @@ export class DonationCompleteComponent implements OnInit {
 
         // Try to patch the person only if they're not already a finalised donor account,
         // e.g. they could have set a password then reloaded this page.
-        if (this.identityService.isTokenForFinalisedUser(idAndJWT.jwt)) {
-          this.registrationComplete = true;
```

Comment:
deleted this line because we don't want to tell the person they are newly registered in case they registered six weeks ago.